admin/controllers/questions.js

```diff
@@ -51,6 +54,18 @@ const getQuestions = async (req, res) => {
 
   // start the check
   try {
+    if (data.pupil.isTestAccount) {
```

Comment:
Do we need to add further checks? E.g. `NODE_ENV !== 'production'`?

Reply:
I've added this to the tech debt backlog and stipulated explicit configuration to enable test accounts.